api/v1/flow/flow.proto

```diff
@@ -21,6 +21,10 @@ message Flow {
     // deprecated in favor of drop_reason_desc.
     uint32 drop_reason = 3 [deprecated=true];
 
+    // auth_type is the authentication type specified for the flow in Cilium Network Policy.
+    // Empty when authentication is not required.
+    string auth_type = 35;
```

Comment:
Should it be a string or an enum?

Reply:
+1 on making this an enum. It's also more efficient on the wire.

Reply:
I was considering both, and concluded that adding new auth methods would be easier if this was a string so that the enum would not need a proto update at that point. Can make this an enum though if that is the consemsus view here :-)

Reply:
Changed to an enum.